src/main/java/com/google/javascript/clutz/DeclarationGenerator.java

```diff
@@ -315,14 +322,18 @@ private boolean isStaticFieldOrMethod(JSType type) {
     return !isClassLike(type) && !isTypedef(type);
   }
 
-  private boolean isDefaultExport(TypedVar symbol) {
-    if (symbol.getType() == null) return false;
-    return !symbol.getType().isObject() ||
-        symbol.getType().isInterface() ||
-        symbol.getType().isInstanceType() ||
-        symbol.getType().isEnumType() ||
-        symbol.getType().isFunctionType() ||
-        isTypedef(symbol.getType());
+  private boolean isDefaultExport(JSType type) {
+    // goog.provides with zero properties are the empty object in closure.
```

Comment:
Maybe extract the comment with the code into its own method? This looks as if it was a comment on `isDefaultExport`, which is confusing.